src/status_im/contexts/wallet/collectible/utils.cljs

```diff
@@ -7,3 +7,14 @@
       first
       :balance
       js/parseInt))
+
+(def ^:private supported-collectible-types
+  #{"image/jpeg"
+    "image/gif"
+    "image/bmp"
+    "image/png"
+    "image/webp"})
+
+(defn collectible-supported?
+  [type]
+  (some #{type} supported-collectible-types))
```

Comment:
We can apply the set as function.

```clojure
(supported-collectible-types type)
```


But I'd suggest renaming `type` toa void shadowing core functions, maybe name it `collectible-type`

Reply:
Done